modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -5,13 +5,13 @@
 
 :_content-type: PROCEDURE
 [id="virt-disable-individual-bootsource-update_{context}"]
-= Disabling an automatic update for a custom boot source
+= Disabling an automatic update for a single boot source
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for a single boot source that has not been customized.
```

Comment:
As before - this is not about customization of a boot resource, but about been in the pre-define list of the boot resources.

This procedure is about disabling the auto-update of a specific pre-defined boot resource, rather than disabling all of them by the feature gate, as described above.

Reply:
Fixed with input from our conversation on Monday, August 1.